docs/reference_guide.md

```diff
@@ -2945,7 +2954,7 @@ Return true if the string haystack contains the string needle, and zero otherwis
 Examples:
 
 ```
-bpftrace -e 't:syscalls:sys_enter_execve /strcontains(str(args->filename),"bin")/ { @[comm, str(args->filename)] = count(); }'  
+bpftrace -e 't:syscalls:sys_enter_execve /strcontains(str(args->filename),"bin")/ { @[comm, str(args->filename)] = count(); }' 
```

Comment:
Same problem as before - please do not update lines unrelated to the PR.

Reply:
My apologies. Still fixing some things...